nevergrad/optimization/recorded_recommendations.csv

```diff
@@ -28,7 +29,7 @@ AvgSmallHaltonSearchPlusMiddlePoint,0.0031863936,0.0076470967,-0.0175068607,0.00
 AvgSmallHammersleySearchPlusMiddlePoint,0.0052440051,-0.0115034938,-0.001397103,0.0084162123,,,,,,,,,,,,
 AvgSmallScrHaltonSearchPlusMiddlePoint,-0.0115034938,0.0122064035,-0.0084162123,0.0106757052,,,,,,,,,,,,
 AvgSmallScrHammersleySearchPlusMiddlePoint,-0.0128155157,0.0,0.004307273,0.0084162123,,,,,,,,,,,,
-BO,-1.7186359043,-0.0719657219,-0.2263452946,-0.0245468071,,,,,,,,,,,,
+BO,-1.1230418754,-0.563634709,2.3536712172,5.1757172963,,,,,,,,,,,,
```

Comment:
There was an update in the bayesian optimization package, and you have been running the old one while the CI uses the new one.
To correct this:
- `pip install -U bayesian-optimization` 
- remove all lines containing BO in this file
- rerun pytest
- commit
Hopefully, all will be good then!

Reply:
Thx, all tests on CI are ok now.